Citadel/Festergut.lua

```diff
@@ -53,6 +54,7 @@ function mod:OnBossEnable()
 end
 
 function mod:OnEngage()
+	self:SetStage(1)
```

Comment:
You don't have to SetStage on WarmUp/OnEngage if the encounter never changes stages :)!